api/src/main/java/ai/djl/repository/Artifact.java

```diff
@@ -319,13 +319,14 @@ public String toString() {
         StringBuilder sb = new StringBuilder(100);
         if (metadata != null) {
             sb.append(metadata.getGroupId())
-                    .append(':')
+                    .append('/')
                     .append(metadata.getArtifactId())
-                    .append(':');
```

Comment:
Doesn't there need to be a trailing '/' here? Otherwise, there wouldn't be a separating slash if you had both artifactId and version